src/coreclr/jit/likelyclass.cpp

```diff
@@ -104,18 +106,18 @@ LikelyClassHistogram::LikelyClassHistogram(INT_PTR* histogramEntries, unsigned e
 }
 
 //------------------------------------------------------------------------
-// getLikelyClass: find class profile data for an IL offset, and return the most likely class
+// getLikelyClasses: find class profile data for an IL offset, and return the most likely classes
 //
 // Arguments:
+//    pLikelyClasses - [OUT] array of likely classes sorted by likelihood (descending)
```

Comment:
It appears that pLikelyClasses is expected to point to an array of `MAX_LIKELY_CLASSES` size on entry. This should be noted in the comment.

However, the API should probably take this as an argument. Maybe there's still a global MAX_LIKELY_CLASSES, but maybe a caller could call this with some smaller 'N', and get the 'N' most likely classes.